src/insights_client/__init__.py

```diff
@@ -399,12 +458,14 @@ class InsightsConstants(object):
         # handle client instantation here so that it isn't done multiple times in __init__
         # The config can be passed now by parameter
         client = InsightsClient(config, False)  # read config, but dont setup logging
+        logger.debug("InsightsClient initialized. Egg version: %s", client.version())
 
-        # we now have access to the clients logging mechanism instead of using print
+        # we now have access to the clients logging mechanism
+        tear_down_logging()
         client.set_up_logging()
-        logging.root.debug("Loaded initial egg: %s", os.path.dirname(insights.__file__))
 
         for p in get_phases():
+            logger.debug("Running phase %s.", p["name"])
```

Comment:
This information is already logged on line 251.

Reply:
I still found it useful, because it’s logged right before entering the _get_phase_ function with its egg selection machinery. `phase '%s'; egg '%s'` is a new message saying that an egg has been selected for the phase.